docs/source/executing-operations/subscription-callback-protocol.mdx

```diff
@@ -195,7 +197,7 @@ Fields and behaviors specific to individual message types are documented below.
 
 During protocol [initialization](#initialization), **Subgraph** sends a synchronous `check` message to **Router** to help ensure that it can send callbacks successfully, and that the `id` and `verifier` fields provided by **Router** are correct.
 
-As long as subscription is active, **Emitter** must send a `check` message to **Router** every five seconds. This enables **Emitter** to confirm both that it can still reach **Router**'s callback endpoint, and that subscription is still active.
+As long as subscription is active, **Emitter** must send a `check` message to **Router** every `heartbeat_interval_ms` milliseconds (value coming from the initial payload sent in `extensions` from the **Router**). This enables **Emitter** to confirm both that it can still reach **Router**'s callback endpoint, and that subscription is still active.
```

Comment:
same comment as above -> do we need to explicitly say that only do it if heartbeats are enabled?